### PR TITLE
[ resolves #5 ] questions models and views added.

### LINKED ISSUE #5: Question content view
Add view for question content with possible answers.
*AC:*
* Design and create model.
* Add simple view rendering the question content with answers in the template.
* Register view under urls.
* Make sure we can manage questions database in Django Admin.
* Question don't need to be "answerable" in this ticket. Will be done in https://github.com/JakMal9/Quiz/issues/6

### PR DESCRIPTION
**Pull request by stanislawK**
resolves #5

### CODE REVIEW

File: quiz_app/questions/models.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+class Question(models.Model):
+    content = models.TextField(unique=True)
+    answers = models.ManyToManyField(Answer, through="QuestionsAnswers")
+
+    def __str__(self):
+        return self.content
+
+
+class QuestionsAnswers(models.Model):
```
Comment (maciejsobolewski) [inline quiz_app/questions/models.py]: Let's stick to one form of naming - singular in this case: `QuestionAnswer`.

File: quiz_app/questions/static/isCorrect.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+const btns = document.querySelectorAll("button");
```
Comment (maciejsobolewski) [inline quiz_app/questions/static/isCorrect.js]: Using js OK, but I suggest to use Django forms for this, just for practice sake.
It should handle validation etc. as well.

File: quiz_app/quiz_app/tests/test_questions_views.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import pytest
```
Comment (maciejsobolewski) [inline quiz_app/quiz_app/tests/test_questions_views.py:1]: We should test not only a happy path, but errors also.
For example: 404 error you catched if the question does not exist, or there is no data in POST request.

**Review by maciejsobolewski — CHANGES_REQUESTED** [review]
Added some initial comments - try to use Django forms for practice.
Kudos for nice tests!

And trifle: remember about newline at the end of the files.

**Comment by maciejsobolewski** [comment]
And the pipelines are working! :)

**Review by maciejsobolewski — APPROVED** [review]

**Review by JakMal9 — APPROVED** [review]

**Comment by maciejsobolewski** [comment]
Idea after testing:
please add the possibility to assign answers to a question in admin question view itself (http://localhost:8000/admin/questions/question/). You can achieve it by adding Django Manager to the models - it's very useful and clean.
You can remove `Question Answers` link completely from the admin after that.

**Review by maciejsobolewski — CHANGES_REQUESTED** [review]
See https://github.com/JakMal9/Quiz/pull/15#issuecomment-1173963056

**Comment by stanislawK** [comment]
> 
Did you mean inline model in admin https://docs.djangoproject.com/en/4.0/ref/contrib/admin/#inlinemodeladmin-objects or modifying our db models to something more abstract with  [manager](https://docs.djangoproject.com/en/4.0/topics/db/managers/), or both together?

**Review by maciejsobolewski — APPROVED** [review]

**Pull request merged**